websites/docs-smoke-test/src/content/components/video-player.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: VideoPlayer component
```

Comment:
I suggest to make that a full width layout page to be able to test (and maybe fix) how it behaves if the page is full width for two-column content on a big screen.